apps/console/src/public/deployment.config.json

```diff
@@ -215,6 +215,17 @@
                     ]
                 }
             },
+            "actions": {
```

Comment:
Send a PR to the `default.json` and also add the `console:actions` in the framework.

Reply:
This is already done.
https://github.com/wso2/carbon-identity-framework/pull/5838